Add Grammalecte

procsynth opened the pull request:
<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks, depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass. 
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can trigger @packagecontrol-bot to re-evaluate your pull request
by pushing a commit or closing and reopening your pull request.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"` 
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.
 
You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists, 
why you believe it is different and needed
below this line. -->

This package enable the usage of Grammalecte, a grammatical checker for french, inside Sublime Text.
This is the only one doing that at the moment.

![screenshot from 2018-05-10 21-26-11](https://user-images.githubusercontent.com/22014799/39892020-8fa346a8-549f-11e8-9b1f-e7dfb198b1b9.png)

Thom1729 commented:
A `is_enabled` that always returns `True` is a no-op.

Rather than use an `EventListener` that always runs, it would be better to use a `ViewEventListener` that only runs for plain text views (or somesuch). This will be especially important when the `on_modified_async` is implemented.

procsynth commented:
Hi,
I've done the modifications you asked for. Those modifications enabled me to implement some more features : https://github.com/procsynth/GrammalecteST3/releases/tag/0.0.3

procsynth commented:
Thank you for taking the time to review my code.

- In the future, the best will be to work with the scope indeed. But I think that change will not be quick. I'd rather prefer for now to listen for events from every view. For now, it will return `True` if the plug-in is running for the current view.
- I believe the default config file is here : https://github.com/procsynth/GrammalecteST3/blob/master/Grammalecte.sublime-settings
- Fixed
- I was not familiar with `enumerate`, fixed, thanks
- Also fixed 
- Got rid of the temp file

With some work, and modification to the libs, I'm now able to use the libs without requiring a shell. So I don't need to run unpacked anymore. I tested, it seems to work out of the shelf.

Thanks for your input.
